src/main/java/org/hl7/davinci/priorauth/Endpoint.java

```diff
@@ -0,0 +1,122 @@
+package org.hl7.davinci.priorauth;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Claim;
+import org.hl7.fhir.r4.model.ClaimResponse;
+import org.hl7.fhir.r4.model.OperationOutcome;
+import org.hl7.fhir.r4.model.OperationOutcome.IssueSeverity;
+import org.hl7.fhir.r4.model.OperationOutcome.IssueType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Endpoint {
+
+    static final Logger logger = LoggerFactory.getLogger(Endpoint.class);
+
+    static final Boolean requestXml = false;
```

Comment:
I think an enumerated type would be better here instead of a pair of booleans. It gets confusing saying requestXml and having it be false.

Reply:
Updated: https://github.com/HL7-DaVinci/prior-auth/commit/19e2506de5c788d626e8365676c4c245a439e052